doc/README.isql_enhancements.txt

```diff
@@ -253,3 +253,74 @@ It requires server v4.0.1 or greater to work.
 Warning: this feature is very tied to engine internals and its usage is discouraged
 if you do not understand very well how these internals are subject to change between
 versions.
+
+
+
+Isql enhancements in Firebird v5.
+---------------------------------
+
+11) SET PER_TABLE_STATS option.
+
+Author: Vladyslav Khorsun <hvlad at users sourceforge net>
+
+When set to ON show per-table run-time statistics after query execution.
+It is set to OFF by default. Also, it is independent of SET STATS option.
+The name PER_TABLE_STATS could be shortened up to PER_TAB. Tables in output
+are sorted by its relation id's.
+
+Example:
+
+-- check current value
+SQL> SET;
+...
+Print per-table stats:   OFF
+...
+
+-- turn per-table stats on
+SQL> SET PER_TABLE_STATS ON;
+SQL>
+SQL> SELECT COUNT(*) FROM RDB$RELATIONS JOIN RDB$RELATION_FIELDS USING (RDB$RELATION_NAME);
+
+                COUNT
+=====================
+                  534
+
+Per table statistics:
+--------------------------------+---------+---------+---------+---------+---------+---------+---------+---------+
+ Table name                     | Natural | Index   | Insert  | Update  | Delete  | Backout | Purge   | Expunge |
+--------------------------------+---------+---------+---------+---------+---------+---------+---------+---------+
+RDB$INDICES                     |         |        3|         |         |         |         |         |         |
+RDB$RELATION_FIELDS             |         |      534|         |         |         |         |         |         |
```

Comment:
Since you have one space in the headers, why not have a space after the numbers?

Reply:
Output is already very wide. Also, I satisfied how it looks even without space after numbers :)